tasmota/tasmota_xdrv_driver/xdrv_57_9_tasmesh.ino

```diff
@@ -546,6 +546,25 @@ void MESHevery50MSecond(void) {
 //          AddLog(LOG_LEVEL_INFO, PSTR("MSH: %*_H), MESH.packetToConsume.front().chunkSize, (uint8_t *)&MESH.packetToConsume.front().payload);
         }
         break;
+      case PACKET_TYPE_HEARTBEAT:
+#if TASMESH_HEARTBEAT
+          for (auto &_peer : MESH.peers){
+            if (memcmp(_peer.MAC, MESH.packetToConsume.front().sender, 6) == 0) {
+              _peer.lastHeartbeatFromPeer = millis();
+
+              if (!_peer.isAlive) {
+                _peer.isAlive = true;
+                char stopic[TOPSZ];
+                GetTopic_P(stopic, TELE, _peer.topic, S_LWT);
+                MqttPublishPayload(stopic, PSTR(MQTT_LWT_ONLINE));
+              }
+              break;
+            }
+          }
+#else
+        break;
```

Comment:
Why don't you leave the `break;` with heartbeat enabled?

Reply:
Missed that, thanks